src/cls/IPM/Utils/Migration.cls

```diff
@@ -5,19 +5,37 @@ Class %IPM.Utils.Migration
 
 ClassMethod RunAll(verbose As %Boolean = 1) As %Status
 {
+    New $Namespace
     Set sc = $$$OK
-    Try {
-        Do ..MigrateZPMToIPM(verbose)
-    } Catch e {
-        Set sc = e.AsStatus()
+    Do ##class(%IPM.Main).GetListNamespace(.list)
```

Comment:
Should run in namespaces with %IPM mapped from the current namespace's default routine database rather than in all namespaces.